packages/studio-ui/src/web/components/Layout/TopNav/cloud/types.ts

```diff
@@ -0,0 +1,13 @@
+import { NLU } from 'botpress/sdk'
+
+/**
+ * CDM Workspace
+ */
+export interface Workspace {
+  id: string
+  name: string
+}
+
+export interface TrainSessions {
```

Comment:
pretty sure this type already exist somewhere

Reply:
Could not find it anywhere

Reply:
I'll check if I can use the type directly from the NLU client

Reply:
Using the NLU client's typings would involve modifying the `TrainingSession` interface [from the SDK](https://github.com/botpress/studio/blob/be32f12148736a4a5243ddd1408f761f82f26155/packages/studio-be/src/sdk/botpress.d.ts#L296). I think this falls out of scope of this PR.